Makefile

```diff
@@ -169,8 +169,8 @@ build-api:
 
 build-web:
 	@echo "++\n***** Building Web for AWS\n++"
-	@yarn workspace @ehpr/web build
-	@yarn workspace @ehpr/web export
+	@NODE_ENV=production && yarn workspace @ehpr/web build
+	@NODE_ENV=production && yarn workspace @ehpr/web export
```

Comment:
This feels a bit sus though. Would you be able to elaborate a bit on what this is doing?

Reply:
These commands should set `NODE_ENV=production` but we've manually set it to `development` at the start of the makefile so it's using that value unless we manually override it again.

We need `NODE_ENV=production` to signal to purge unused CSS among other likely optimizations